### PR TITLE
RLS: v0.13.0

### PR DESCRIPTION
**Pull request by cpcloud**

### CODE REVIEW

**Comment by cpcloud** [comment]
@conda-forge-admin, please rerender

**Comment by cpcloud** [comment]
@jakirkham @ocefpaf Is there some reason appveyor builds are failing on platforms that have a version of miniconda installed later than 4.3? A downgrade to a compatible version (4.3.x) is attempted but the job fails:

https://ci.appveyor.com/project/cpcloud/ibis-framework-feedstock/build/job/dg173u27aa9x09ql

**Comment by ocefpaf** [comment]
> Is there some reason appveyor builds are failing on platforms that have a version of miniconda installed later than 4.3?

The re-re-rendering should take care of that.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,51 +5,51 @@ package:
...
-    - python-graphviz
-    - python-hdfs >=2.0.0  # [not py36]
+    - regex
     - setuptools
     - six
-    - sqlalchemy >=1.0.0
-    - thrift  # [py36]
-    - thrift <=0.9.3  # [not py36]
-    - thriftpy <=0.3.9
     - toolz
```
Comment (jakirkham) [inline recipe/meta.yaml:27]: Would encourage you to drop everything from `build` that is not needed at build time and re-render again.

Comment (cpcloud) [reply recipe/meta.yaml:27]: Ok great, will give that a try. Thank you.

**Comment by cpcloud** [comment]
@conda-forge-admin, please rerender

**Pull request merged**